.github/workflows/release.yaml

```diff
@@ -106,10 +106,6 @@ jobs:
       - if: matrix.target == 'x86_64-unknown-linux-gnu'
         run: rm -rf editors/code/server
 
-      - if: matrix.target == 'x86_64-unknown-linux-gnu'
-        run: npx vsce package -o ../../dist/rust-analyzer.vsix
```

Comment:
Maybe call it `rust-analyzer-fallback.vsix` or `rust-analyzer-unsupported-platform`? By the way do I see it correctly that the extension doesn't get a fallback for unsupported platforms published to the vscode marketplace?

Reply:
> By the way do I see it correctly that the extension doesn't get a fallback for unsupported platforms published to the vscode marketplace?

Yeah, I don't know what happens if we try to publish one (and I won't be the one who tries it).

Reply:
One problem is that you need a matching server binary (in principle), and if we don't publish the extension for a platform, we don't publish a server binary either.

Reply:
I would expect it to be easier for people to build their own rust-analyzer server than their own extension as the later requires nodejs. While the former only requires rustc which anyone using rust-analyzer probably already has installed anyway.

Reply:
Yeah, fair enough.